src/libs/fileDownload/index.js

```diff
@@ -0,0 +1,39 @@
+import openURLInNewTab from '../openURLInNewTab';
+import getAttachmentName from './getAttachmentName';
+
+/**
+ * Downloading attachment in web, desktop
+ * @param {String} url
+ */
+export default function fileDownload(url) {
+    fetch(url)
+        .then(response => response.blob())
+        .then((blob) => {
+            // Create blob link to download
+            const href = URL.createObjectURL(
+                new Blob([blob]),
+            );
+
+            // creating anchor tag to initiate download
+            const link = document.createElement('a');
```

Comment:
I was really confused here and thought "there's no way this is the best way to download something in 2021, right?" but after doing some searching uh ya I guess it still is 🤯 https://davidwalsh.name/javascript-download

Reply:
Yes right?
I was also searching for a better way to do that. But found this is still valid.